### PR TITLE
Improve dispatcher doc

### LINKED ISSUE #112: Add documentation about Openstack Object Store Service Key
In the current doc version of the Dispatcher component, it does not explain very well about the key used to generate links to the data stored in the Object Store. Improve the documentation so that you can configure it correctly.

### PR DESCRIPTION
**Pull request by thiagoyeds**
Fix #112

### CODE REVIEW

File: config/dispatcher.conf [hunk 1]
```diff
@@ -35,7 +35,7 @@ openstack_user_password=$openstack_user_password
 openstack_object_store_service_api_url=$openstack_object_store_service_api_url
 # Openstack object store service container name
 openstack_object_store_service_container_name=$openstack_object_store_service_container_name
-# Openstack object store service key
+# Openstack object store service key (random 8 chars string)
```
Comment (thiagomanel) [inline config/dispatcher.conf]: replace by `Openstack object store service key`

File: docs/dispacher-install.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ mvn install -Dmaven.test.skip=true
 
 Edit the files:
 - [Dispatcher configuration file](/config/dispatcher.conf) to allow its communication with the SAPS Catalog and Permanent Storage.
+- The ```$openstack_object_store_service_key``` argument in [Dispatcher configuration file](/config/dispatcher.conf) used to access Object Store must be configured using the command ```swift post -m “Temp-Url-Key:$openstack_object_store_service_key”``` passing as other information as parameters. Use a random 8 chars string and run the following command to configure:
```
Comment (thiagomanel) [inline docs/dispacher-install.md]: replace by
```
- The $openstack_object_store_service_key property in [Dispatcher configuration file](/config/dispatcher.conf) is used to access Object Store. After creating a new key, it must be configured using the command ```swift post -m “Temp-Url-Key:$openstack_object_store_service_key”
```

File: docs/dispacher-install.md [hunk 3]
```diff
@@ -30,6 +30,14 @@ mvn install -Dmaven.test.skip=true
...
+
+```bash
+swift post -m "Temp-URL-Key:$openstack_object_store_service_key" --os-auth-url $openstack_identity_service_api_url --auth-version 3 --os-user-id $openstack_user_id --os-password $openstack_user_password --os-project-id $openstack_project_id
+```
+
+Note: ```-auth-version``` is version in operation of the Openstack Identity Service API
```
Comment (thiagomanel) [inline docs/dispacher-install.md]: replace by

Note: ```-auth-version``` is the version of the deployed Openstack Identity Service API

**Review by thiagomanel — CHANGES_REQUESTED** [review]
thanks, @thiagoyeds  

I've added three small requests

File: config/dispatcher.conf (hunk 1 above)
Comment (thiagoyeds) [reply config/dispatcher.conf]: Done

File: docs/dispacher-install.md (hunk 2 above)
Comment (thiagoyeds) [reply docs/dispacher-install.md]: Done

File: docs/dispacher-install.md (hunk 3 above)
Comment (thiagoyeds) [reply docs/dispacher-install.md]: Done

**Review by thiagomanel — APPROVED** [review]
lgtm. Thanks, @thiagoyeds

**Pull request merged**